cli/cli_app.go

```diff
@@ -528,6 +528,9 @@ func runTerraformWithRetry(terragruntOptions *options.TerragruntOptions) error {
 	// Retry the command configurable time with sleep in between
 	for i := 0; i < terragruntOptions.MaxRetryAttempts; i++ {
 		if out, tferr := shell.RunTerraformCommandWithOutput(terragruntOptions, terragruntOptions.TerraformCliArgs...); tferr != nil {
+			if out == nil {
+				return tferr
+			}
```

Comment:
NIT: might be cleaner below to say `if out != nil && isRetryable(...)`

Reply:
Ah good point! Fixed: 76bfb0c